src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/FundingComplete.cshtml

```diff
@@ -0,0 +1,22 @@
+﻿@model SFA.DAS.EmployerAccounts.Web.ViewModels.AccountDashboardViewModel
+
+@if (Model.ReservedFundingToShow != null)
```

Comment:
Please avoid the 'if' statement in the view if possible.  May be a case for two partials here.  One for when waiting and the other when the event has been received.  The waiting one could be reused elsewhere, the logic on what partial to use should be in the controller.